src/v2/components/CopyableCorrelationId.tsx

```diff
@@ -0,0 +1,43 @@
+import { Check, Copy, X } from "@phosphor-icons/react"
```

Comment:
Let's use the SSR icons here ;) (just change the import to `@phosphor-icons/react/dist/ssr` & specify the `weight="bold"` attribute on the icons if needed)